spacewalk/config/spacewalk-config.spec

```
@@ -206,5 +207,12 @@ if [ -e /etc/pki/tls/private/uyuni.key ]; then
   fi
 fi
 
+if [[ ! -f /etc/pki/trust/anchors/LOCAL-RHN-ORG-TRUSTED-SSL-CERT ]] ; then
```

```suggestion
if [ ! -f /etc/pki/trust/anchors/LOCAL-RHN-ORG-TRUSTED-SSL-CERT ] ; then
```

One should be enough. 2 may have already a different meaning.